SampleApp/SampleApp/MessageCenterViewController.xaml

```diff
@@ -0,0 +1,36 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<ContentPage
+    Title="{x:Static resources:AppResources.message_center_title}"
+    xmlns:resources="clr-namespace:SampleApp"
+    xmlns="http://xamarin.com/schemas/2014/forms"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml" x:Class="SampleApp.MessageCenterViewController">
+    <ContentPage.Content>
+        <ListView x:Name="listView" ItemSelected="listView_ItemSelected" RowHeight="70">
+            <ListView.ItemTemplate>
+                <DataTemplate>
+                    <ViewCell>
+                        <StackLayout Orientation="Horizontal" Padding="20">
+                            <!-- To be used once we add the IsRead attribute-->
```

Comment:
Are you looking for `bool Unread { get; }` ?

Reply:
For the moment it's not available in the **Message** object. I don't know when it will be planned. @rlepinski do we need to add it know with my PR?

Reply:
I don't think that work will be done before we release this version. I'd rather that you remove the commented out code before merging, we can always add it back later once that feature is available.